config/formats.js

```diff
@@ -704,7 +708,7 @@ exports.Formats = [
 	},
 	{
 		name: "[Gen 6] Battle Factory",
-
+      desc: [Randomized teams of Pokémon for a generated Smogon tier with sets that are competitively viable.],
```

Comment:
There is an extra space before `desc` again, the sentence needs to be surrounded in quotes, the `e`  in `Pokemon` needs to be escaped, and there needs to be a blank line after this line.